cachito/web/content_manifest.py

```diff
@@ -232,3 +238,205 @@ def generate_icm(self, image_contents=None):
         icm = deepcopy(BASE_ICM)
         icm["image_contents"] = image_contents or []
         return deep_sort_icm(icm)
+
+
+@dataclass(frozen=True)
+class Package:
```

Comment:
What do you think to have separate `Package` and `Dependency` classes? IMO, this separation could give a clear model design of the packages and dependencies used in Cachito.

Reply:
I totally agree, I believe we can follow the design you posted on a previous comment.